tests/cli/test_schema_generation.py

```diff
@@ -126,7 +126,9 @@ class SubPipelineComponentCorrectDocstr(SubPipelineComponent):
 MODULE = EmptyPipelineComponent.__module__
 
 
-@pytest.mark.filterwarnings("ignore:handlers", "ignore:config", "ignore:enrich")
+@pytest.mark.filterwarnings(
+    "ignore:handlers", "ignore:config", "ignore:enrich", "ignore:validate_name"
```

Comment:
what does this do?

Reply:
Schema generation logs warnings when a class is skipped and I found it annoying to constantly see the warnings in pytest, so I suppressed them.